packages/app/src/specs/banners/CreateOrganizationBanner.vue

```diff
@@ -8,6 +8,12 @@
     class="mb-16px"
     :icon="OrganizationIcon"
     dismissible
+    :has-banner-been-shown="hasBannerBeenShown"
+    :event-data="{
+      campaign: 'Set up your organization',
+      medium: 'Specs Create Organization Banner',
+      cohort: '' // TODO Connect cohort
```

Comment:
What's the trigger for this to get completed? Will it be done as part of the existing cohort work after this is merged? It would be good to add some context/link to the issue. Though I think your are coordinating with @warrensplayer already.

Reply:
Correct, @warrensplayer is working on the Cohort assignment & persistence logic under #23083 and will hook these up as part of that work

Reply:
I think it's #23083